lib/logstash/inputs/beats.rb

```diff
@@ -2,21 +2,16 @@
 require "logstash/inputs/base"
 require "logstash/namespace"
 require "logstash/timestamp"
-require "lumberjack/beats"
-require "lumberjack/beats/server"
 require "logstash/codecs/identity_map_codec"
 require "logstash/codecs/multiline"
-require "logstash/inputs/beats_support/circuit_breaker"
-require "logstash/inputs/beats_support/codec_callback_listener"
-require "logstash/inputs/beats_support/connection_handler"
-require "logstash/inputs/beats_support/event_transform_common"
-require "logstash/inputs/beats_support/decoded_event_transform"
-require "logstash/inputs/beats_support/raw_event_transform"
-require "logstash/inputs/beats_support/synchronous_queue_with_offer"
 require "logstash/util"
-require "thread_safe"
+require "vendor/jar-dependencies/logstash-input-beats-3.1.0.beta1.jar"
```

Comment:
This causes Logstash to fail to load the plugin because the file is not found because it's the wrong path.

The .gemspec puts `vendor/jar-dependencies` into ruby's library path, so you can just do `require "logstash-input-beats-3.1.0.beta1.jar"` here. If I make this change, I can load this plugin into logstash.